services/app/src/pages/game/Game.jsx

```diff
@@ -43,15 +56,14 @@ function Room() {
       <CopyToClipboard text={roomUrl}>
         <button type="button">Copy link</button>
       </CopyToClipboard>
-      <Player name={username} />
-      <div>
-        <h2>Team</h2>
-        <div>
-          {users.map(
-            (user) => user.id !== findUserByUsername(username).id && user.name,
-          )}
-        </div>
-      </div>
+      <Player user={findUserByUsername(username)} skipMove={skipMove} />
+      <Team
+        title="Team"
+        users={users.filter(
```

Comment:
Filtering users array will cause a lot of rerenders of `Team` component as it will return a new array on every render.

I would suggest to use `useMemo` here to reduce re-renders here:
```
  const currentUser = useMemo(
    () => users.find(user => user.name === userName),
    [users]
  );
  const team = useMemo(
    () => users.filter(user => user.id !== currentUser.id),
    [users, currentUser]
  ); 
```

Reply:
At this point when the game component is only dependant on users it seems that is not worth to use the useMemo hook.